spk/mono/src/service-setup.sh

```diff
@@ -1,7 +1,7 @@
 
 # mono service setup
 
-service_postinst ()
+service_prestart ()
 {
     # Sync ca certificates
     ${SYNOPKG_PKGDEST}/bin/cert-sync /etc/ssl/certs/ca-certificates.crt
```

Comment:
I still need to find out if this works with DSM7 as I'm not 100% sure about the permissions

Reply:
A new release would also fix this, but I think this is better in the long run.

Reply:
What was I thinking. mono isn't a daemon.